packages/web-app-files/tests/unit/components/FilesList/ResourceTable.spec.ts

```diff
@@ -259,9 +270,43 @@ describe('ResourceTable', () => {
       expect(wrapper.classes()).toContain('oc-table-hover')
     })
   })
+
+  describe('tags', () => {
+    it('renders more than 2 tags and more button', () => {
```

Comment:
description is wrong, the cell doesn't render more than 2 tags. It renders up to 2 tags (and a more button if there are more than 2 tags).

Reply:
```suggestion
    it('renders 2 tags and more button if tag count greater than 2', () => {
```

Reply:
Could you be more fine grained in the `tags` section with nested `describe`s? The following test structure could make sense:

```
describe('tags',{
  describe('more tags button', {
    it.each.todo( ... tag count 0 has no more button, tag count 1 has no more button, tag count 2 has no more button, tag count 3 has more button, tag count 57 has more button ...),
    it.todo('opens sidebar on clicking more')
  }
  describe('inline tags', { 
    it.todo('has no inline tags if tag count 0'),
    it.todo('has one inline tag if tag count 1'),
    it.todo('has two inline tags if tag count 2'),
    it.todo('has two inline tags if tag count 57')
  }
}
```